src/component/user/UserProfileStart.jsx

```diff
@@ -51,7 +66,7 @@ const UserProfileStart = () => {
               Dietary Preference
             </Typography>
             <Typography variant="body1">
-              {logStatus.dietaryPrefs || 'No preferences, a food enjoyer'}
+              {displayPrefs}
```

Comment:
did we lose the default value here?

Reply:
Oh yeah, let me add the default value in